app/frontend/components/pages/sendAda/sendAdaPage.tsx

```diff
@@ -45,23 +46,21 @@ const SendValidation = ({sendFormValidationError, txSuccessTab}) =>
     )
   )
 
+type Searchable = {
+  searchable: {
+    ticker: string
+    assetNameReadable: string
```

Comment:
I'd consider distinguishing rather assetNameHex and assetNameUtf8 in the codebase to make the respective encodings clearer to the reader which is in the end what matters when working with the data and may otherwise not be that obvious

Reply:
Refactored